### PR TITLE
#23 #22 improve ux for the continuous annotation task for clarity

### LINKED ISSUE #23: Improve UX for the continuous annotation task for clarity
The continuous annotation task UI has been mostly finalized, but the following elements need to be taking into account:
- Make it clearer that you can as a test run press S to try it 
- Simplify a bit the participant not found 
- Add video is finished, please continue 
- Do a pop up to say that the annotation will be re-written 
- Change name to "Redo annotation " instead of "start annotating"? 
- Make big comment that if the person move out of view, to change the view 
- Remove the submission code and change font to make it fit better with the instructions 
- Prefilter participants by 1080p

### PR DESCRIPTION
**Pull request by Kenneth-Funes-TU-Delft**
- Improved instructions about practicing with the annotation key before
- Implemented logic to hide the instructions bar on the left and increase the size of the video player annotate
- Created a separate component for the Gallery and the participant image crop
- Created a separate component for the "Instructions sidebar" so the main file is cleaner
- Created a separate component for the TaskProgress so the main file is cleaner
- Created a separate component for the flashscreen
- Minor style modifications, such as reducing the width of the right sidebar

Fixes #23 #22

### CODE REVIEW

File: covfee/client/journey/journey.tsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
-import * as React from "react"
-import styled from "styled-components"
-import { generatePath } from "react-router"
 import { ArrowRightOutlined, PlusOutlined } from "@ant-design/icons"
-import { Row, Col, Typography, Menu, Button, Modal, Progress } from "antd"
+import { Button, Menu, Modal, Progress, Row } from "antd"
```
Comment (Era-Dorta) [inline covfee/client/journey/journey.tsx:2]: There should be no changes in this file, I think it'd be better to undo them and then merge #38 into this branch

File: covfee/client/tasks/continuous_annotation/conflab_participant_selection.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useRef } from "react"
+// You can download this image and place it in the art folder from:
+// @covfee.ewi.tudelft.nl:/home/kfunesmora/conflab-media/
```
Comment (Era-Dorta) [inline covfee/client/tasks/continuous_annotation/conflab_participant_selection.tsx:3]: I don't think you want to commit this path, same below

**Review by Era-Dorta — APPROVED** [review]

File: covfee/client/tasks/continuous_annotation/conflab_participant_selection.tsx (hunk 2 above)
Comment (Kenneth-Funes-TU-Delft) [reply covfee/client/tasks/continuous_annotation/conflab_participant_selection.tsx:3]: Ok. I won't. My assumption was that access is restricted anyway and there isn't much info given there.

File: covfee/client/journey/journey.tsx (hunk 1 above)
Comment (Kenneth-Funes-TU-Delft) [reply covfee/client/journey/journey.tsx:2]: You are right. I'll undo them, and hopefully #38 and this PR will get merged at the same time.

Comment (Kenneth-Funes-TU-Delft) [reply covfee/client/journey/journey.tsx]: Changes in journey.tsx were removed.

**Pull request merged**